bentoml/_internal/configuration/containers.py

```diff
@@ -410,6 +427,29 @@ def tracer_provider(
                 agent_host_name=jaeger_server_address,
                 agent_port=jaeger_server_port,
             )
+            provider.add_span_processor(BatchSpanProcessor(exporter))  # type: ignore (no opentelemetry types)
+            return provider
+        elif (
+            tracer_type == "otlp"
+            and otlp_server_protocol is not None
+            and otlp_server_url is not None
+        ):
+            if otlp_server_protocol == "grpc":
+                # pylint: disable=no-name-in-module # https://github.com/open-telemetry/opentelemetry-python-contrib/issues/290
+                from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import (
+                    OTLPSpanExporter,  # type: ignore (no opentelemetry types)
```

Comment:
remove type: ignore here? I think by default we enable pyright to infer from library code.

Reply:
Same, got it from lines 408 and 423.